app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] I got an error when running the seed. This is because here on line 8 you are writing `likes_counter` while in your schema you have `LikesCounter`. Kindly fix this

Reply:
This has been resolved.